resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/File.java

```diff
@@ -299,6 +300,17 @@ public InputStream getContents(boolean force) throws CoreException {
 		return getLocalManager().read(this, force, null);
 	}
 
+	/** like {@link #getContents(boolean)} **/
+	@Override
+	public byte[] readAllBytes() throws CoreException {
+		boolean force = true;
```

Comment:
Inline?

Reply:
that is outlined to have symmetry with org.eclipse.core.internal.resources.File.getContents(boolean)